iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,8 @@
 
 /* Begin PBXBuildFile section */
 		443FCD9426137A4C009D6DE6 /* Frames in Frameworks */ = {isa = PBXBuildFile; productRef = 443FCD9326137A4C009D6DE6 /* Frames */; };
+		95B343DA2853582D001AAA44 /* CustomStyle2.swift in Sources */ = {isa = PBXBuildFile; fileRef = 95B343D92853582D001AAA44 /* CustomStyle2.swift */; };
```

Comment:
Nice one , You will probably have to add to carthage example app as well, I had only tested using pod example.